resources/views/settings/security/2fa/setup.blade.php

```diff
@@ -22,19 +22,19 @@
 			  	<div class="collapse show" id="step1">
 				  	<p>You will need to install a compatible mobile app, we recommend the following apps:</p>
 				  	<ul>
-				  		<li><a href="https://1password.com/downloads/" rel="nooopener nofollow">1Password</a></li>
-				  		<li><a href="https://authy.com/download/" rel="nooopener nofollow">Authy</a></li>
-				  		<li><a href="https://lastpass.com/auth/" rel="nooopener nofollow">LastPass Authenticator</a></li>
+				  		<li><a href="https://1password.com/downloads/" rel="noopener nofollow">1Password</a></li>
+				  		<li><a href="https://authy.com/download/" rel="noopener nofollow">Authy</a></li>
+				  		<li><a href="https://lastpass.com/auth/" rel="noopener nofollow">LastPass Authenticator</a></li>
```

Comment:
this is a bug fix... https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/noopener